integration-tests/goss/windows/commands/validate.goss.yaml

```diff
@@ -0,0 +1,12 @@
+---
+# TODO: coverage for the add {test} permutations
+command:
+  "validate":
+    exit-status: 0
+    exec: "release/goss-alpha-windows-amd64 --use-alpha=1 -g integration-tests/goss/windows/commands/validate-input.yaml validate"
+    stdout:
```

Comment:
Hah, you did it! Using goss to test goss :)

Reply:
It seemed like a good idea ;)